diff --git a/net/minecraft/block/state/BlockStateBase.java b/net/minecraft/block/state/BlockStateBase.java
index ead7331..7f799e5 100644
--- a/net/minecraft/block/state/BlockStateBase.java
+++ b/net/minecraft/block/state/BlockStateBase.java
@@ -25,7 +25,7 @@
       }
 
       private <T extends Comparable<T>> String getPropertyName(IProperty<T> var1, Comparable<?> var2) {
-         return var1.getName(var2);
+         return var1.getName((T)var2);
       }
    };
 
@@ -33,7 +33,9 @@
    }
 
    public <T extends Comparable<T>> IBlockState cycleProperty(IProperty<T> var1) {
-      return this.withProperty(var1, (Comparable)cyclePropertyValue(var1.getAllowedValues(), this.getValue(var1)));
+      //return withProperty(var1, (Comparable)cyclePropertyValue(var1.getAllowedValues(), this
+      //        .getValue(var1)));
+      return withProperty(var1, cyclePropertyValue(var1.getAllowedValues(), this.getValue(var1)));
    }
 
    protected static <T> T cyclePropertyValue(Collection<T> var0, T var1) {
