diff --git a/net/minecraft/entity/Entity.java b/net/minecraft/entity/Entity.java
index a4ed686..ac54151 100644
--- a/net/minecraft/entity/Entity.java
+++ b/net/minecraft/entity/Entity.java
@@ -2161,7 +2161,7 @@
    private <T extends Entity> void getRecursivePassengersByType(Class<T> var1, Set<T> var2) {
       for(Entity var4 : this.getPassengers()) {
          if(var1.isAssignableFrom(var4.getClass())) {
-            var2.add(var4);
+            var2.add((T) var4);
          }
 
          var4.getRecursivePassengersByType(var1, var2);
