diff --git a/net/minecraft/util/IntHashMap.java b/net/minecraft/util/IntHashMap.java
index ac68ef4..edbef2e 100644
--- a/net/minecraft/util/IntHashMap.java
+++ b/net/minecraft/util/IntHashMap.java
@@ -26,7 +26,7 @@
 
       for(IntHashMap.Entry var3 = this.slots[getSlotIndex(var2, this.slots.length)]; var3 != null; var3 = var3.nextEntry) {
          if(var3.hashEntry == var1) {
-            return var3.valueEntry;
+            return (V)var3.valueEntry;
          }
       }
 
